src/Razor/src/Microsoft.CodeAnalysis.Remote.Razor/UriPresentation/RemoteUriPresentationService.cs

```diff
@@ -37,7 +37,11 @@ protected override IRemoteUriPresentationService CreateService(in ServiceArgs ar
             context => GetPresentationAsync(context, span, uris, cancellationToken),
             cancellationToken);
 
-    private async ValueTask<TextChange?> GetPresentationAsync(
+    private static IRemoteUriPresentationService.Response ShouldCallHtml => new(ShouldCallHtml: true, TextChange: null);
```

Comment:
Nit: it is slightly odd to have a should and don't wording. Maybe `CallHtml` and `DontCallHtml`

Reply:
Fair. What also annoys me is that the "Dont" part is always flagged as a spelling mistake :)

Reply:
Went with "CallHtml" and "NoFurtherHandling". No typos!